netbox/dcim/api/serializers.py

```diff
@@ -714,6 +714,7 @@ class VirtualDeviceContextSerializer(NetBoxModelSerializer):
     primary_ip = NestedIPAddressSerializer(read_only=True, allow_null=True)
     primary_ip4 = NestedIPAddressSerializer(required=False, allow_null=True)
     primary_ip6 = NestedIPAddressSerializer(required=False, allow_null=True)
+    status = ChoiceField(choices=VirtualDeviceContextStatusChoices)
```

Comment:
I think this needs `required=False` to prevent it from being mandatory for PATCH requests.

```suggestion
    status = ChoiceField(choices=VirtualDeviceContextStatusChoices, required=False)
```

Reply:
Will that not make POST requests optional? Because the field on the model does not have any default value.

Reply:
Disregard: I tested it out and it works fine as-is.